client/my-sites/plans/woo-express-plans-page/index.tsx

```diff
@@ -11,20 +11,19 @@ import {
 } from '@automattic/calypso-products';
 import page from '@automattic/calypso-router';
 import { Button, Card } from '@automattic/components';
-import { Plans, type SiteDetails } from '@automattic/data-stores';
+import { Plans, type SiteDetails, SitePlan } from '@automattic/data-stores';
 import { formatCurrency } from '@automattic/format-currency';
 import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
 import { useCallback } from 'react';
 import BodySectionCssClass from 'calypso/layout/body-section-css-class';
-import { SitePlanData } from 'calypso/my-sites/checkout/src/hooks/product-variants';
```

Comment:
Kinda weird where this lives, perhaps even why it even exists in the shape/form that it does